### PR TITLE
[FSPE-5176] Improved border utility colors

### PR DESCRIPTION
**Pull request by mattnickles**
- Border utility now supports the color transparent to hide an elements border without altering its size. Requirement for portal.
- Border utility documentation now includes usage guidelines for each example.
- Added variables for Border utility colors.

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://e6b7b99a6f2559c3aa4073a0fa52224c15b0baee-chi-pr.enterprise.ctl.io" target="_blank"> https://e6b7b99a6f2559c3aa4073a0fa52224c15b0baee-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1086/ for more details.

File: src/website/views/utilities/border.pug [hunk 1]
```diff
@@ -85,29 +100,69 @@ p.-text
...
 p.-text
-  | Breakpoint classes also work here. The format is <code>-b[-{breakpoint}]--{color}</code>.
+  | Use breakpoint classes to change an elements border color on different viewports. Follow the format <code>-b[-{breakpoint}]--{color}</code>.
 .example.-mb--3
   .-p--2
     .chi-grid.-no-gutter.-text.-text--center
-      .-m--2.-p--5.-b--1.-b--primary.-b-sm--dark.-b-md--danger.-b-lg--success.-b-xl--warning(style="background:#F8F8F8") -b--primary -b-sm--dark -b-md--danger -b-lg--info -b-xl--warning
+      .chi-col.-w--8
+        .-m--1.-p--1.-py--4.-b--1.-b--primary.-b-sm--dark.-b-md--danger.-b-lg--success.-b-xl--warning.-bg--grey-10
+          | -b--primary -b-sm--dark -b-md--danger -b-lg--info -b-xl--warning
```
Comment (jllr) [inline src/website/views/utilities/border.pug:166]:
```suggestion
          | -b--primary -b-sm--dark -b-md--danger -b-lg--success -b-xl--warning
```

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://344be025d52f2bd1d3e9ec42e33bb264bc647fa9-chi-pr.enterprise.ctl.io" target="_blank"> https://344be025d52f2bd1d3e9ec42e33bb264bc647fa9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1090/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-qugii6, https://ops.elasticbox.com/services/instances/i-zklid0

**Pull request merged**